crates/ruff_python_formatter/src/auxiliary/mod.rs

```diff
@@ -0,0 +1,5 @@
+pub mod expression;
```

Comment:
What's your motivation for putting all formatting logic into the `auxiliary` module?

Reply:
no specific one, should i unpack the auxiliary module at the top level instead?

Reply:
I would be in favour of removing, or renaming the folder to format if you prefer to have all of them in a single folder (I'm okay with manually adding the group modules to lib)